msprime/simulations.py

```diff
@@ -963,6 +964,72 @@ def mean_recombination_rate(self):
 
         return np.average(rates, weights=weights)
 
+    def slice(self, start=None, end=None, remove_flanks=False):
```

Comment:
Actually, can we rename this parameter to ``trim`` please? This would make it consistent with some tskit functions that do similar things, e.g. https://tskit.readthedocs.io/en/latest/python-api.html#tskit.TreeSequence.ltrim

Reply:
Yup. slice is now trim, so 'slice' just means python slice syntax.

Reply:
Sorry Graham, I should have been more explicit here, I meant to change the ``remove_flanks`` parameter to ``trim``, so the signature would be ``slice(start=None, end=None, trim=False)``

Reply:
n'th time's a charm... :)

Reply:
:+1: